test/e2e/framework/util.go

```diff
@@ -380,6 +382,55 @@ func RunIfSystemSpecNameIs(names ...string) {
 	skipInternalf(1, "Skipped because system spec name %q is not in %v", TestContext.SystemSpecName, names)
 }
 
+// Run a test container to try and contact the Kubernetes api-server from a pod, wait for it
+// to flip to Ready, log its output and delete it.
+func runKubernetesServiceTestContainer(c clientset.Interface, ns string) {
```

Comment:
I am not sure whether it is appropriate to put this into `util.go`, as we are still cleaning it. 
/cc @andrewsykim

Reply:
I agree it's not ideal but seems okay because we're at least removing it from `test/e2e/e2e.go`. Any suggestions where else it should go?

Reply:
How about putting this in `initialize.go` (or `setup.go`) ?

Reply:
It's a very generic function, to me it doesn't make sense there.

Reply:
Interesting that this code actually lived in `util.go` and it was moved on https://github.com/kubernetes/kubernetes/commit/79202656bc34ecde65a3abb45dc54ef4ec480342